test/manual/s3_test.cc

```diff
@@ -61,3 +61,58 @@ SEASTAR_TEST_CASE(test_client_put_get_object) {
         cln->close().get();
     });
```

Comment:
"Putting a large object into S3 using plain PUT is bad choice -- one need
to collect the whole object in memory, then send it as a content-length
request with plain body. "

not always. Some sstable components such as Statistics.DB and TOC.txt will always be small (smaller than 8k), therefore they're good candidates for s3 PUT interface.

Reply:
Ah you said large. Forget it :-)